apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -4069,7 +4073,7 @@
       "tr": "Turkish",
       "ja": "Japanese",
       "ko": "Korean",
-      "pt" : "Português (Brasil)"
```

Comment:
![kill-it-with-fire-fire](https://user-images.githubusercontent.com/91890529/225570280-01d33250-439c-4b85-82f3-716cbc78533c.gif)